db/migrate/20240110025234_add_columns_to_users.rb

```diff
@@ -0,0 +1,19 @@
+class AddColumnsToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :role, :string
+    add_column :users, :first_name, :string
+    add_column :users, :middle_name, :string
+    add_column :users, :last_name, :string
+    add_column :users, :username, :string
+    add_column :users, :location, :string
+    add_column :users, :company, :string
+    add_column :users, :job_title, :string
+    add_column :users, :profile_img, :text
+    add_column :users, :bio, :text
+    add_column :users, :guidance, :text, array: true, default: []
+    add_column :users, :language, :text, array: true, default: []
+    add_column :users, :skill, :text, array: true, default: []
+    add_column :users, :isAvailable, :string
```

Comment:
```suggestion
 add_column :users, :is_available, :string
```
isAvailable is in camel case, let's use snake case.